src/bookshelf/src/bksw_conf.erl

```diff
@@ -114,46 +123,45 @@ reset_dispatch() ->
 dispatch() ->
     {keys, {AccessKeyId, SecretAccessKey}} = keys(),
     Config = [{stream_download, stream_download()},
+              {auth_check_disabled, auth_check_disabled()},
               {access_key_id, AccessKeyId},
-              {secret_access_key, SecretAccessKey}],
+              {secret_access_key, SecretAccessKey}|sql_retry()],
```

Comment:
It appears that we're setting the sql portion of the config in 3 different ways in this file:

1. explicitly on lines 56/57
2. via `lists:flatten` on line 70
3. and here on line 128 via head / tail construction